pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/codestyle/xml/UnnecessaryLocalBeforeBranch.xml

```diff
@@ -65,7 +65,7 @@ public class Foo {
     </test-code>
 
     <test-code>
-        <description>#1495 [java] UnnecessaryLocalBeforeReturn with assert</description>
+        <description>#1495 [java] UnnecessaryLocalBeforeBranch with assert</description>
```

Comment:
Well, we usually don't rename such old test cases, as the github issue still is about the rule, that was named at its time "UnnecessaryLocalBeforeReturn".

Don't use "find and replace everywhere"...

If we are really going to rename the rule, we would document it in the rule description, that this rule was named "UnnecessaryLocalBeforeReturn" until PMD version x.y.z ("previously known as ...").